integration-tests/emulated/chains/relays/polkadot/src/lib.rs

```diff
@@ -21,7 +21,7 @@ use emulated_integration_tests_common::{
 	impl_hrmp_channels_helpers_for_relay_chain, impl_send_transact_helpers_for_relay_chain,
 	xcm_emulator::decl_test_relay_chains,
 };
-use polkadot_primitives::runtime_api::runtime_decl_for_parachain_host::ParachainHostV5;
```

Comment:
I think there is no strong reason not to bump to version 10 right away? Are there any other node features which are enabled via runtime API version?

Also with the release on Polkadot, we should be able to bump the base version [here](https://github.com/paritytech/polkadot-sdk/blob/a84dd0dba58d51503b8942360aa4fb30a5a96af5/polkadot/primitives/src/runtime_api.rs#L131).

Reply:
Put differently, I believe the blocker for bumping the API was asynchronous backing. If that blocker is removed, I am not aware of anything holding back the remaining versions.

Reply:
```
		/// Returns a list of all disabled validators at the given block.
		#[api_version(8)]
		fn disabled_validators() -> Vec<ValidatorIndex>;

		/***** Added in v9 *****/

		/// Get node features.
		/// This is a staging method! Do not use on production runtimes!
		#[api_version(9)]
		fn node_features() -> vstaging::NodeFeatures;

		/***** Added in v10 *****/
		/// Approval voting configuration parameters
		#[api_version(10)]
		fn approval_voting_params() -> ApprovalVotingParams;
```

These are the other, node_features and approval_voting_params are 0 risk, for `disabled_validators` I will let @ordian and @tdimitrov answer.

Reply:
disabled validators should also not do anything. Just exposing the API for later use.

Reply:
Done.